react/features/base/conference/functions.js

```diff
@@ -73,6 +73,7 @@ export function commonUserJoinedHandling(
     } else {
         dispatch(participantJoined({
             botType: user.getBotType(),
+            connectionStatus: user.getConnectionStatus(),
```

Comment:
does this make sense? I suppose it would be active or undefined, since they just joined?

Reply:
Yep. I  think it always returns ACTIVE for now. I was wondering whether to just put ACTIVE but at the end I decided that this decision should be left to lib-jitsi-meet...